contents/blog/session-replay-pricing.md

```diff
@@ -0,0 +1,94 @@
+---
+date: 2024-07-22
+title: "We’ve decided to make less money [Part 1]"
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/take_my_money_11433edb48.png
+author:
+  - andy-vandervell
+featuredImageType: full
+tags:
+  - PostHog news
+---
+
+## TL;DR
+ 
+- We’ve optimized our session replay pipeline and infrastructure.
+
+- We’re passing the savings onto you by making session replay _drastically cheaper_, especially at lower volumes.
+
+- We’re now the cheapest session replay tool among competitive products.
+
+And this is just Part 1...
+
+## How will this impact me?
+
+It depends on usage, but a customer generating 25k recorded sessions a month will now pay $85 a month, down from $430 a month. That's a huge $4,140 annual saving.
```

Comment:
```suggestion
It depends on your usage, but as an example: if you capture 25k session recordings per month your bill will now be $85 a month, down from $430 a month. That's a huge $4,140 annual saving.
```

You don't 'generate recorded sessions' - you 'capture session replays/recordings'. I also think this is nicer if we frame it around the user as a person and not just say 'a customer'